### PR TITLE
data: shubhsharma19

### PR DESCRIPTION
**Pull request by shubhsharma19**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2749"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There is a missing comma `,`

File: data/shubhsharma19.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    {
+      "name": "Interested in reading my blogs? Here you go!",
+      "url": "https://shubhsharma19.hashnode.dev/",
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/shubhsharma19",
+      "icon": "FaLinkedin"
+    }
```
Comment (Pradumnasaraf) [inline data/shubhsharma19.json:34]:
```suggestion
    },
```

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by shubhsharma19** [comment]
Thanks a lot, @Pradumnasaraf for the help. 😄

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**